database/migrations/2025_09_05_054824_create_predefined_filters_permissions.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('predefined_filter_permissions', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('predefined_filter_id')
+                ->constrained('predefined_filters')
+                ->onDelete('cascade');
+            
+            $table->unsignedBigInteger('permission_group_id');
+            $table->unsignedInteger('created_by');
+            $table->timestamps();
```

Comment:
I don't know if the should keep the timestamps in the predefined_filter_permissions table. If we remove it `public $timestamps = false;` needs to be added in the model.